docs/src/00-doc/04_release_management.stories.mdx

```diff
@@ -22,4 +22,19 @@ Design System changes are supposed to influence application styles, therefore a
 ## Breaking changes
 
 Deciding if a change to the Design System is "breaking" is the responsibility of the developer(s) making the change.  
-It is also their responsibility to inform other relevant roles like UX and PM (if the change is being made at their request) of that, so the additional effort of updating all consuming applications can be taken into consideration.
\ No newline at end of file
+It is also their responsibility to inform other relevant roles like UX and PM (if the change is being made at their request) of that, so the additional effort of updating all consuming applications can be taken into consideration.
+
+## Creating a release 
+
+Once your changes are merged and you want to release a new version of the components library you need to take the following steps:
+ - To release a new version run the command `lerna version [major | minor | patch | premajor | preminor | prepatch | prerelease]`. This will take of updating the version number, committing and pushing the changes.
```

Comment:
```suggestion
 - To release a new version run the command `lerna version [major | minor | patch | premajor | preminor | prepatch | prerelease]`. This will take care of updating the version number, committing and pushing the changes.
```

Reply:
done